src/main/java/frc/robot/ClampedController.java

```diff
@@ -24,11 +24,20 @@ public ClampedController(double p, double minPower, double maxPower) {
 
 	@Override
 	public double calculate(double currentValue, double setpoint) {
-		return clamp(super.calculate(currentValue, setpoint));
+		double power = clamp(super.calculate(currentValue));
+		super.setSetpoint(setpoint);
+		if (super.atSetpoint()) {
+			power = 0;
+		}
+		return power;
```

Comment:
This feels off

Reply:
Its not

Reply:
This is important for drive PIDs that control multiple things at once, it negates the min speed when close, very important